### PR TITLE
Add to travis PHP 7.4nightly and try to run tests on PHP 7.4 and nightly

### PR DESCRIPTION
**Pull request by pawelkg**
Fix issue #2293

**Description**
Travis configuration.

**Checklist:**
- [X] Securely signed commits
- [X] Component(s) with PHPdocs
- [X] Unit testing, with >80% coverage
- [X] User guide updated
- [X] Conforms to style guide

### CODE REVIEW

**Comment by jim-parry** [comment]
ould this be done as two matrices, one for PHP7.2 & 7.3 and one for the others?
Your approach is likely to get unwieldly as we add more database drivers

**Comment by pawelkg** [comment]
I don't think there is much we can do. While I'm not expert in Travis config file, I prepared some simplifies.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
+        - DB=sqlite
+    - php: 7.4snapshot
+      env:
+        - EXTENDED=false
+        - DB=mysqli
+    - php: 7.4snapshot
+      env:
+        - EXTENDED=false
+        - DB=postgres
+    - php: 7.4snapshot
```
Comment (samsonasik) [inline .travis.yml]: there is already stable `7.4` in travis

Comment (pawelkg) [reply .travis.yml]: I update my PR

**Comment by lonnieezell** [comment]
I'm not a fan of going to a full matrix currently, honestly. I do think we should add 7.4 in right away (I'll take care of that momentarily.) It's always possible we get another solution in the place in the near future. Closing.

**Pull request closed without merging**